tests/unit/verticals/sca/test_file_selection.py

```diff
@@ -39,6 +46,32 @@ def test_get_all_files_from_sca_paths(tmp_path):
     assert Path("foo/bar/file4.json") in [Path(filepath) for filepath in files]
 
 
+def test_get_ignored_files(tmp_path, cli_fs_runner: CliRunner):
+    """
+    GIVEN a directory
+    WHEN calling sca scan a directory
+    THEN certain ignored binary files should be omitted
+    """
+
+    tmp_paths = [str(tmp_path / filename) for filename in BINARY_NAMES]
+    for path in tmp_paths:
+        write_text(filename=path, content="")
+
+    with cd(tmp_path):
+        result = cli_fs_runner.invoke(
+            cli,
+            ["sca", "scan", "all", ".", "--verbose"],
+        )
+
+        pattern = r"^ignoring binary file: (.+)$"
+        matches = re.findall(pattern, result.output, re.MULTILINE)
+        matches_paths = [Path(match) for match in matches]
+
+        assert Path("file3.png") in matches_paths
+        assert Path("front/file1.png") in matches_paths
+        assert Path(".git/file2.png") not in matches_paths
```

Comment:
Could you test more granularly ? By invoking `get_filepaths` or `is_filepath_excluded` for instance ?
Here the test rely on the pattern of a verbose message which is not very reliable (even if it test the global behavior we want to match)